python/paddle/fluid/tests/unittests/test_elementwise_max_op.py

```diff
@@ -46,6 +49,50 @@ def test_check_grad_ingore_y(self):
             ['X'], 'Out', max_relative_error=0.005, no_grad_set=set('Y'))
 
 
+# Get cuda version from command line
+def get_cuda_runtime_version():
+    command = os.popen("nvcc --version").read()
+    command = re.split(' ', command)
+    tag = 0
+    for _ in command:
+        tag += 1
+        if _ == 'release':
+            break
+    return int(float(command[tag].replace(',', '')) * 1000)
+
+
+@unittest.skipIf(
+    core.is_compiled_with_cuda() and get_cuda_runtime_version() < 11000,
```

Comment:
not core.is_compiled_with_cuda() or core.cudnn_version() < 8100

Reply:
Because support cpu bf16 kernel, so change to "core.is_compiled_with_cuda() and core.cudnn_version() < 8100"